subprojects/docs/src/docs/release/notes.md

```diff
@@ -112,6 +112,10 @@ The kind folks at [Algolia](https://www.algolia.com/) kindly host an index used
 You can now specify a timeout duration for a task, after which it will be interrupted.
 Read more [about task timeouts](userguide/more_about_tasks.html#sec:task_timeouts) in the docs.
 
+## HTTP retries
```

Comment:
Okay, looking at the [the commit](https://github.com/gradle/gradle/commit/156b5ebd00971cc7fd6bd94420f0fa63b5488e8d) this seems to only be used for dependency management (not, say, remote scripts). 

Notable things to add to the documentation are: 
 * 3 tries total (2 re-tries) before a repo is blacklisted
 * starting with a 125ms delay
 * This is not configurable

Reply:
I suggest making the title reflect that this is dependency management specific

Reply:
I updated the rest of it. Can I merge this or do you want any more changes?